src/org/exist/start/start.config

```diff
@@ -104,10 +104,4 @@ tools/jetty/lib/*                                  mode == standalone
 tools/jetty/lib/*                                  mode == other
 tools/ircbot/lib/*                                 mode == jetty
 tools/aspectj/lib/*                                always
-tools/yajsw/wrapperApp.jar                         mode == other
-tools/yajsw/wrapper.jar                            mode == other
-tools/yajsw/lib/extended/velocity/velocity-%latest%.jar         mode == other
-tools/yajsw/lib/core/commons/commons-configuration2-%latest%.jar mode == other
-tools/yajsw/lib/core/commons/commons-lang3-%latest%.jar          mode == other
-tools/yajsw/lib/core/netty/netty-all-4.0.35.Final.jar           mode == other
-tools/yajsw/lib/core/jna/jna-platform-%latest%.jar              mode == other
+tools/yajsw/lib/core/commons/commons-configuration2-%latest%.jar mode == other
```

Comment:
Apologies, I thought you had added a YAJSW dependency on commons-configuration2 to start.config. I see now that it was already there and just moved. The problem was there before your change.

The problem is that `LauncherWrapper.java` which is part of the core of exist (i.e. it is compiled into `exist.jar`), has both a compile-time and run-time dependency on commons-configuration2. However, that dependency is not present in `lib/core`, it is also missing from the IDE files and also from the Maven POM. Copying the file into `lib/core` and adding the missing declarations will fix it (#1494), alongside updating the pom.xml for exist-core.